src/utilities/util_verify.c

```diff
@@ -17,11 +17,13 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
     size_t size;
     int ch;
     char *config, *dump_offsets, *uri;
-    bool dump_address, dump_blocks, dump_history, dump_layout, dump_pages, stable_timestamp;
+    bool dump_address, dump_blocks, dump_layout, dump_pages, dump_history, hs_verify,
+      stable_timestamp;
 
-    dump_address = dump_blocks = dump_history = dump_layout = dump_pages = stable_timestamp = false;
+    dump_address = dump_blocks = dump_history = dump_layout = dump_pages = hs_verify =
+      stable_timestamp = false;
     config = dump_offsets = uri = NULL;
-    while ((ch = __wt_getopt(progname, argc, argv, "d:S")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "d:S:h")) != EOF)
```

Comment:
Options should appear in sorted order, `d:hS:`

Reply:
The colon here denotes that a parameter should follow, e.g. `d:` is `-d dump_timestamps`. I added `-S` with no parameters so there should be no colon following S e.g. `d:hS

Reply:
Sorry about that Jeremy, thanks!